docs/content/preview/api/ysql/the-sql-language/statements/dml_move.md

```diff
@@ -11,12 +11,12 @@ type: docs
 ---
 
 {{< warning title="YSQL currently supports only fetching rows from a cursor consecutively in the forward direction." >}}
-See the subsection [Beware Issue #6514](../../../cursors/#beware-issue-6514) in the generic section [Cursors](../../../cursors/).
+See the subsection [Beware Issue #6514](../../../cursors/#beware-issue-6514) in the generic section [Cursors](../../../cursors/). In particular, every `MOVE` variant causes the _0A000_ with a message like "MOVE not supported yet".
```

Comment:
"causes the _0A000_ " ... the what? error?

Reply:
Anybody who knows PostgreSQL or who uses YSQL will know what this means. See the PG Doc on [Error Codes](https://www.postgresql.org/docs/11/errcodes-appendix.html)